components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/cache/SessionContextCache.java

```diff
@@ -42,11 +45,17 @@ public class SessionContextCache extends BaseCache<SessionContextCacheKey, Sessi
 
     private static final String SESSION_CONTEXT_CACHE_NAME = "AppAuthFrameworkSessionContextCache";
     private static final Log log = LogFactory.getLog(SessionContextCache.class);
+    private Boolean isSessionDataStorageOptimizationEnabled = true;
```

Comment:
Is there any specific reason have `Boolean` object, rather than the primitive type?

Reply:
Try to make this a final variable.

Reply:
Fixed with [`c43c465`](https://github.com/wso2/carbon-identity-framework/pull/4446/commits/c43c465fb4b8edc33bf309a076d8e8226b5df9e1)